_themes/launcher.md

```diff
@@ -0,0 +1,18 @@
+---
+heading: Launcher
+thumbnail: themes/images/thumbnails/launcher.jpg
```

Comment:
You haven't actually *included* these screenshots in your PR.  I can walk you through creating them if you'd like, or I can make them for you this one time. :wink:

I've been using Firefox's "Responsive Design Mode" to get the screenshot sizes right. (1440x900 for fullsize, 1080x880 for thumbnails, but then scaled down to 270x220).  I've also been saving them as 85% quality jpg for a little size optimization.

Up to you how you want to handle it.  I can provide the screenshots if necessary, but you submit Themes so much it'd be appreciated if you can provide your own. :smile: